Services/User/Profile/classes/class.ilPersonalProfileGUI.php

```diff
@@ -527,7 +533,124 @@ protected function showUserAgreement()
         $this->tpl->setPermanentLink('usr', null, 'agreement');
         $this->tpl->printToStdout();
     }
-    
+
+    protected function showConsentWithdrawalConfirmation() : void
+    {
+        $this->tabs->clearTargets();
+        $this->tabs->clearSubTabs();
+        $this->tpl->setTitle($this->lng->txt('withdraw_consent'));
+
+        $defaultAuth = AUTH_LOCAL;
+        if ($this->setting->get('auth_mode')) {
+            $defaultAuth = $this->setting->get('auth_mode');
+        }
+
+        $tpl = new \ilTemplate('tpl.withdraw_terms_of_service.html', true, true, 'Services/TermsOfService');
+        /** @var ilObjUser $user */
+        $user = $GLOBALS['DIC']->user();
+        if ( $user->getAuthMode() == AUTH_LDAP
+            || ( $user->getAuthMode() == 'default' && $defaultAuth == AUTH_LDAP) )
+        {
+            $message = nl2br(
+                $this->lng->txt('withdrawal_mail_info')
+                . $this->lng->txt('withdrawal_mail_text')
+            );
+
+            $tpl->setVariable('TERMS_OF_SERVICE_WITHDRAWAL_CONTENT', $message);
+            $tpl->setVariable('FORM_ACTION', $this->ctrl->getFormAction($this,'cmd[withdrawAcceptanceLDAP]'));
```

Comment:
The 2nd argument is wrong:

```suggestion
            $tpl->setVariable('FORM_ACTION', $this->ctrl->getFormAction($this, 'withdrawAcceptanceLDAP'));
```